architect/databases/postgresql/partition.py

```diff
@@ -103,7 +105,7 @@ def prepare(self):
 
         return self.database.execute(execute_sql.format(**definitions).format(
             pk=' AND '.join('{pk} = NEW.{pk}'.format(pk=pk) for pk in self.pks),
-            parent_table=self.table,
+            parent_table=f"{schema if schema else 'public'}.{self.table}",
```

Comment:
We should add the schema prefix to specific places in the command, and it's ok to use `public` if not given